modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc

```diff
@@ -294,21 +233,18 @@ IMPORTANT NOTES:
     Donating to EFF:                    https://eff.org/donate-le
 ----
 
-You can see that the SSL certificate’s been successfully created, and
-that it will expire on 2018-06-18.
+You can see that the SSL certificate’s been successfully created, and that it will expire on 2018-06-18.
```

Comment:
```suggestion
You can see that the SSL certificate has been successfully created and that it will expire on 2018-06-18.
```

Reply:
Maybe use a date in the future from now although that will be outdated again in a couple of months?